src/core/ext/transport/chttp2/server/chttp2_server.cc

```diff
@@ -1144,3 +1201,31 @@ void grpc_server_add_channel_from_fd(grpc_server* /* server */, int /* fd */,
 }
 
 #endif  // GPR_SUPPORT_CHANNELS_FROM_FD
+
+absl::Status grpc_server_add_passive_listener(
+    grpc_core::Server* server, grpc_server_credentials* credentials,
+    std::shared_ptr<grpc_core::experimental::PassiveListenerImpl>
+        passive_listener) {
+  grpc_core::ExecCtx exec_ctx;
+  GRPC_API_TRACE("grpc_server_add_passive_listener(server=%p, credentials=%p)",
+                 2, (server, credentials));
+  // Create security context.
+  if (credentials == nullptr) {
+    return GRPC_ERROR_CREATE("No credentials specified for passive listener");
```

Comment:
Let's use `absl::UnavailableError()` instead of `GRPC_ERROR_CREATE()` here.

Same on line 1218.

Reply:
Done. I didn't realize we were fully deprecating the function. SGTM.